src/xenia/app/emulator_window.h

```diff
@@ -95,9 +95,6 @@ class EmulatorWindow {
 
   void ToggleProfilesConfigDialog();
   void SetHotkeysState(bool enabled) { disable_hotkeys_ = !enabled; }
-  // We need to store it somewhere so there will be no situation when there are
-  // multiple instances opened.
-  std::unique_ptr<CreateProfileDialog> profile_creation_dialog_;
```

Comment:
Not necessary anymore due to profile creation dialog being changed into popup modal. Popup modal blocks all input behind it.